Makefile-plugins.mk

```diff
@@ -89,9 +89,28 @@ $(pGen)/%.go: $(MYGOBIN)/pluginator $(MYGOBIN)/goimports
 		$(MYGOBIN)/goimports -w $*.go \
 	)
 
-# Target is for debugging.
+# Generate builtin plugins
 .PHONY: generate-kustomize-builtin-plugins
-generate-kustomize-builtin-plugins: $(builtinplugins)
+generate-kustomize-builtin-plugins: $(builtplugins)
```

Comment:
Maybe I think it is better to separate a step of generate and a step of diff check.
Because `generate-kustomize-builtin-plugins` will be used in developing built-in plugins. This command may error when developing plugins before exec git commit.
I think it better to separate them there, It will be useful for developing plugins.

Reply:
I see your point. Do you know if diff check will be called anywhere else? Since the context is strictly limited to builtin plugins maybe I should use `builtin-plugins no diff` as target name instead

Reply:
I imagine making a new `builtin-plugins diff` command, and that makes called from [here](https://github.com/kubernetes-sigs/kustomize/pull/5303/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R112).
I'll leave it to you to figure out how to do it.